systemtest/src/main/java/io/strimzi/systemtest/performance/report/parser/PerformanceMetricsParser.java

```diff
@@ -0,0 +1,392 @@
+                bobUpdateTimes.append(Double.parseDouble(simpleMetrics.get(key).split(" ")[0]) / 1000.0);
+            }
+        }
+        return bobUpdateTimes.toString();
+    }
+
+
+    private static void printRow(String[] row, int[] columnWidths) {
+        StringBuilder sb = new StringBuilder("|");
+        for (int i = 0; i < row.length; i++) {
+            sb.append(String.format(" %-" + columnWidths[i] + "s |", row[i]));
+        }
+        System.out.println(sb);
+    }
+
+    private static void printSeparator(int[] columnWidths) {
+        StringBuilder sb = new StringBuilder("+");
+        for (int width : columnWidths) {
+            sb.append("-".repeat(width + 2)).append("+"); // +2 for the padding on either side of the value
+        }
+        System.out.println(sb);
+    }
+
+    private static String[] getHeadersForUseCase(String useCaseName) {
+        return switch (useCaseName) {
+            case PerformanceConstants.TOPIC_OPERATOR_ALICE_BULK_USE_CASE -> new String[]{
+                "Experiment", PerformanceConstants.NUMBER_OF_TOPICS, PerformanceConstants.NUMBER_OF_CLIENT_INSTANCES,
+                PerformanceConstants.NUMBER_OF_MESSAGES, PerformanceConstants.CREATION_TIME + " (s)",
+                PerformanceConstants.SEND_AND_RECV_TIME + " (s)", PerformanceConstants.DELETION_TIME + " (s)",
+                PerformanceConstants.TOTAL_TEST_TIME + " (s)",
+                "STRIMZI_MAX_BATCH_SIZE", "MAX_BATCH_LINGER_MS", "Reconciliation Max Duration (s)",
+                "Max Batch Size", "Max Queue Size", "UTO Event Queue Time (s)",
+                "Describe Configs Max Duration (s)", "Create Topics Max Duration (s)",
+                "Reconciliations Max Duration (s)", "Update Status Max Duration (s)",
+                "Max System Load Average 1m Per Core (%)", "Max JVM Memory Used (MBs)"
+            };
+            case PerformanceConstants.TOPIC_OPERATOR_BOBS_STREAMING_USE_CASE -> new String[]{
+                "Experiment", PerformanceConstants.NUMBER_OF_TOPICS, PerformanceConstants.NUMBER_OF_CLIENT_INSTANCES,
+                PerformanceConstants.NUMBER_OF_MESSAGES, "Bob Update Times Round 1,2,3 (s)",
+                PerformanceConstants.BOB_NUMBER_OF_TOPICS_TO_UPDATE,
+                PerformanceConstants.CREATION_TIME + " (s)",
+                PerformanceConstants.SEND_AND_RECV_TIME + " (s)", PerformanceConstants.DELETION_TIME + " (s)",
+                PerformanceConstants.TOTAL_TEST_TIME + " (s)",
+                "STRIMZI_MAX_BATCH_SIZE", "MAX_BATCH_LINGER_MS", "Reconciliation Max Duration (s)",
+                "Max Batch Size", "Max Queue Size", "UTO Event Queue Time (s)",
+                "Describe Configs Max Duration (s)", "Create Topics Max Duration (s)",
+                "Reconciliations Max Duration (s)", "Update Status Max Duration (s)",
+                "Max System Load Average 1m Per Core (%)", "Max JVM Memory Used (MBs)"
+            };
```

Comment:
Wouldn't be better to have these documented somewhere else, instead of creating Strings based on case?
Because once you will have more and more test cases, I don't really want to see how much messy it will become.

Reply:
+1

Reply:
Okay, now it's generic :) [1]

[1] - https://github.com/strimzi/strimzi-kafka-operator/pull/9907/commits/09c805278dff9965128df8b63aed3a18f8362770